rspack.config.js

```diff
@@ -221,6 +222,7 @@ const config = {
         process.env.MB_EDITION === "ee"
           ? ENTERPRISE_SRC_PATH + "/overrides"
           : SRC_PATH + "/lib/noop",
+      embedding: EMBEDDING_SRC_PATH,
```

Comment:
this is needed otherwise, rspack wouldn't be able to resolve this new directory since it's at the same level as `embedding-sdk`